### PR TITLE
add tool bar to bottom of article

### PR DESCRIPTION
**Pull request by andygnewman**
- repeats the article toolbar (share, comments, print, save) at the bottom of the article.
- tracking will differentiate actions taken between top and bottom.

### CODE REVIEW

File: client/components/share/main.js [hunk 1]
```diff
@@ -4,16 +4,17 @@ const OShare = require('o-share');
...
-		let url = shareCount.getAttribute('data-shared-url');
+	if(shareCountArray) {
+		let url = shareCountArray[0].getAttribute('data-shared-url');
 		if(url && url.length) {
 			fetch(`/article/${articleId}/social-counts?url=${url}`, { credentials: 'same-origin' })
 			.then(fetchres.json)
 			.then(function(counts) {
-				shareCount.textContent = `${counts.shares}  shares`;
+				shareCountArray.forEach(shareCount => shareCount.textContent = `${counts.shares}  shares`)
 			});
```
Comment (i-like-robots) [inline client/components/share/main.js]: Some squiffy formatting

File: client/components/share/main.js [hunk 2]
```diff
@@ -4,16 +4,17 @@ const OShare = require('o-share');
...
 
 function loadShareCount() {
-	let shareCount = document.querySelector('.js-share-count');
+	let shareCountList = document.querySelectorAll('.js-share-count');
+	let shareCountArray = Array.prototype.slice.call(shareCountList);
 	let article = document.querySelector('.article');
 	let articleId = article.getAttribute('data-content-id');
-	if(shareCount) {
-		let url = shareCount.getAttribute('data-shared-url');
+	if(shareCountArray) {
```
Comment (i-like-robots) [inline client/components/share/main.js]: check length, empty array will be truthy

File: client/components/share/main.scss [hunk 3]
```diff
@@ -40,6 +40,9 @@
 	}
 
 	&__print {
+		@include oGridRespondTo(M) {
```
Comment (i-like-robots) [inline client/components/share/main.scss]: Linter should really enforce any mixins or extends to be below properties

File: views/partials/social.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<div class="article__share" data-trackable="share">
+<div class="article__share {{#ifEquals location 'bottom'}}article__share--bottom{{/ifEquals}}" data-trackable="share-{{location}}">
```
Comment (i-like-robots) [inline views/partials/social.html]: just have `article__share--{{location}}` for consistency

**Comment by i-like-robots** [comment]
Looks good, couple of minor things

File: client/components/share/main.js (hunk 2 above)
Comment (andygnewman) [inline client/components/share/main.js]: I spotted that just after committing.

**Pull request merged**